tests/Browser/BlogManageTest.php

```diff
@@ -0,0 +1,530 @@
+<?php
+
+namespace Tests\Browser;
+
+use Laravel\Dusk\Browser;
+use Tests\DuskTestCase;
+use Illuminate\Foundation\Testing\DatabaseMigrations;
+
+class BlogManageTest extends DuskTestCase
+{   
+    use DatabaseMigrations;
+
+    public function setUp() :void
+    {
+        parent::setUp();
+        $this->artisan('db:seed --class=UserTableTestAddUserSeeder');
+        $this->artisan('db:seed --class=BlogTableTestAddBlogSeeder');
+    }
+    
+    /**
+     * Test login blog with user can see blog.
+     * case 1.1: Test screen Blog list
+     * 
+     * @return void
+     */
+    public function testLoginBlog()
+    {
+        $user = $this->userCanSee();
+        $this->browse(function ($browser) use ($user) {
+            $browser->visit('/login')
```

Comment:
In test file, '/login', '/blog',... re-use multiple time.
Why don't make constant for it and call this constant.
Refer:
https://github.com/loitq/training/blob/f8051435a4240b760a72b9369680d74c0f31585f/config/constants.php#L4-L7

Reply:
I fixed